contrib/debian/rules

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/make -f
+
+# Find the arch we are building for, as this determines
+# the location of plugins in /usr/lib
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
+%:
+	# For jessie and beyond
+	dh $@ --with systemd
+	# For wheezy use the following (and also remove
+	# dh-systemd Build-Depends from the control file)
+	#dh $@
+
+override_dh_auto_configure:
+	# Build the configure script if it doesn't already exist
+	-[ ! -x $(CURDIR)/configure ] && $(CURDIR)/autogen.sh
```

Comment:
@terryburton yes works fine - apart from when building with sbuild you have to have dh-autoreconf installed on the main system as well, and if it isn't it just bombs out with an error. But 99.9% of people won't be doing that, so it shouldn't matter :)

@lhw confirmed - extra dependencies can be removed. Thanks.